.gitignore

```diff
@@ -1,3 +1,9 @@
+.idea/
```

Comment:
This file will have a conflict once `server` gets merged. We will have to wait for #11 to check in.